examples/multiple_data_source_examples/public/components/data_source_list_active_example.tsx

```diff
@@ -0,0 +1,166 @@
+ */
+import React from 'react';
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { MountPoint, CoreStart } from 'opensearch-dashboards/public';
+import {
+  DataSourceAggregatedViewConfig,
+  DataSourceManagementPluginSetup,
+} from 'src/plugins/data_source_management/public';
+import { ComponentProp } from './types';
+
+interface DataSourceListActiveExampleProps {
+  savedObjects: CoreStart['savedObjects'];
+  dataSourceEnabled: boolean;
+  notifications: CoreStart['notifications'];
+  setActionMenu?: (menuMount: MountPoint | undefined) => void;
+  dataSourceManagement: DataSourceManagementPluginSetup;
+}
+
+export const DataSourceListActiveExample = ({
+  savedObjects,
+  dataSourceEnabled,
+  notifications,
+  setActionMenu,
+  dataSourceManagement,
+}: DataSourceListActiveExampleProps) => {
+  const DataSourceMenu = dataSourceManagement.ui.getDataSourceMenu<
+    DataSourceAggregatedViewConfig
+  >();
+  const data: ComponentProp[] = [
+    {
+      name: 'savedObjects',
+      required: true,
+      defaultValue: '-',
+      description: 'The saved object client is used to fetch available data sources',
+      deprecated: false,
+    },
+    {
+      name: 'notifications',
+      required: true,
+      defaultValue: '-',
+      description: 'The notifications toasts object exposes interfaces to show toasts',
+      deprecated: false,
+    },
+    {
+      name: 'hideLocalCluster',
+      required: false,
+      defaultValue: 'false',
+      description: 'The property to hide local cluster from the data source selector',
+      deprecated: false,
+    },
+    {
+      name: 'fullWidth',
+      required: true,
+      defaultValue: '-',
+      description:
+        'The property of OutComboBox and when specified to true would expand to the entire width available',
+      deprecated: false,
+    },
+    {
+      name: 'displayAllCompatibleDataSources',
+      required: false,
+      defaultValue: 'undefined',
+      description: 'When specified to true, it will show all compatible data sources',
+      deprecated: false,
+    },
+    {
+      name: 'dataSourceFilter',
+      required: false,
+      defaultValue: 'undefined',
+      description:
+        'When specified, the filter function will be used to filter the available options before rendering',
+      deprecated: false,
+    },
+    {
+      name: 'activeDataSourceIds',
+      required: false,
+      defaultValue: 'undefined',
+      description:
+        'This property is only assessed when displayAllCompatibleDataSources is set to false, it is used to specify the active data source ids and the component will show active data sources',
+      deprecated: false,
+    },
+  ];
+
+  const columns: Array<EuiBasicTableColumn<ComponentProp>> = [
+    {
+      field: 'name', // need to match the type
```

Comment:
`// need to match the type` should match the property name in the ComponentProp type?

Reply:
Removed the comment in new commit